### PR TITLE
add config import.memory-use-ratio

### PR DESCRIPTION
**Pull request by joccau**
Signed-off-by: joccau <zak.zhao@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
...
 
+### `memory-use-ratio` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+> **注意：**
+>
+> 从 v6.5.0 起，PITR 恢复功能支持直接将备份日志文件读取到内存中做恢复，此配置项主要指定 TiKV 中用于 PITR 恢复的内存使用占比。
+
++ 用于 PITR 日志恢复功能，表示 PITR 恢复中可用内存与系统总内存的占比。
++ 可调整范围：[0.0, 0.5]
++ 默认值：0.3，表示系统 30% 的内存可用于 PITR 恢复；当为 0.0 时，表示使用 download 日志文件的模式做 PITR 恢复。
```
Comment (Leavrth) [inline tikv-configuration-file.md:1616]: 文档有提到过 download 日志文件模式嘛？没有的话写先下载到硬盘之类的通俗的话可能会好一些

**Review by Leavrth — APPROVED** [review]
rest LGTM

File: tikv-configuration-file.md (hunk 1 above)
Comment (joccau) [reply tikv-configuration-file.md]: 已改

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
 + 默认值：8
 + 最小值：1
 
+### `memory-use-ratio` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+> **注意：**
+>
+> 在小于 v6.5.0 的版本中，PITR 恢复仅支持将备份文件下载到本地的模式做恢复，从 v6.5.0 开始，PITR 功能支持直接读取备份日志文件到内存中做恢复，此配置项主要指定 TiKV 中用于 PITR 恢复的内存使用占比。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1612]:
```suggestion
> 在小于 v6.5.0 的版本中，PITR 仅支持将备份文件下载到本地进行恢复。从 v6.5.0 开始，PITR 支持直接读取备份日志文件，然后在内存中进行恢复。此配置项指定 TiKV 中用于 PITR 恢复的内存使用占比。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
...
+
++ 用于 PITR 日志恢复功能，表示 PITR 恢复中可用内存与系统总内存的占比。
++ 可调整范围：[0.0, 0.5]
++ 默认值：0.3，表示系统 30% 的内存可用于 PITR 恢复；当为 0.0 时，表示使用下载日志文件到本地的模式做 PITR 恢复。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1616]:
```suggestion
+ 默认值：`0.3`，表示系统 30% 的内存可用于 PITR 恢复；当为 `0.0` 时，表示通过下载日志文件到本地进行 PITR 恢复。
```

**Review by shichun-0415 — APPROVED** [review]

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
 + 默认值：8
 + 最小值：1
 
+### `memory-use-ratio` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+> **注意：**
+>
+> 在小于 v6.5.0 的版本中，PITR 仅支持将备份文件下载到本地进行恢复。从 v6.5.0 开始，PITR 支持直接读取备份日志文件，然后在内存中进行恢复。此配置项指定 TiKV 中用于 PITR 恢复的内存使用占比。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1612]:
```suggestion
> 在小于 v6.5.0 的版本中，PITR 仅支持将备份文件下载到本地进行恢复。从 v6.5.0 开始，PITR 支持直接读取内存中的备份日志文件，然后进行恢复。此配置项指定 TiKV 中用于 PITR 恢复的内存使用占比。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
...
 
+### `memory-use-ratio` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+> **注意：**
+>
+> 在小于 v6.5.0 的版本中，PITR 仅支持将备份文件下载到本地进行恢复。从 v6.5.0 开始，PITR 支持直接将备份日志文件读取到缓存中，然后进行恢复。此配置项指定 TiKV 中用于 PITR 恢复的内存使用占比。
+
++ 用于 PITR 日志恢复功能，表示 PITR 恢复中可用内存与系统总内存的占比。
++ 可调整范围：[0.0, 0.5]
++ 默认值：`0.3`，表示系统 30% 的内存可用于 PITR 恢复；当为 `0.0` 时，表示通过下载日志文件到本地进行 PITR 恢复。
```
Comment (hfxsd) [inline tikv-configuration-file.md:1616]:
```suggestion
+ 从 v6.5.0 开始，PITR 支持直接将备份日志文件读取到缓存中，然后进行恢复。此配置项用于 PITR 日志恢复功能，表示 PITR 恢复中可用内存与系统总内存的占比。
+ 可调整范围：[0.0, 0.5]
+ 默认值：`0.3`，表示系统 30% 的内存可用于 PITR 恢复；当为 `0.0` 时，表示通过下载日志文件到本地进行 PITR 恢复。

> **注意：**
>
> 在小于 v6.5.0 的版本中，PITR 仅支持将备份文件下载到本地进行恢复。
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1605,6 +1605,16 @@ Raft Engine 相关的配置项。
 + 默认值：8
 + 最小值：1
 
+### `memory-use-ratio` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
++ 从 v6.5.0 开始，PITR 支持直接将备份日志文件读取到缓存中，然后进行恢复。此配置项用于 PITR 日志恢复功能，表示 PITR 恢复中可用内存与系统总内存的占比。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1610]:
```suggestion
+ 从 v6.5.0 开始，PITR 支持直接将备份日志文件读取到缓存中，然后进行恢复。此配置项用来配置 PITR 恢复中可用内存与系统总内存的占比。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**